server/src/main/scala/com/cloudera/livy/LivyConf.scala

```diff
@@ -109,6 +109,10 @@ object LivyConf {
   // How often Livy polls YARN to refresh YARN app state.
   val YARN_POLL_INTERVAL = Entry("livy.server.yarn.poll-interval", "5s")
 
+  // Http compression is enabled or not, if true the json data transmitted with be
+  // compressed with GZIP.
+  val COMPRESS_SUPPORTED = Entry("livy.server.content_compress.enabled", false)
```

Comment:
I think `GZipSupport` looks at `Accept-Header`. What do you think about enabling `GZipSupport` all the time? Then we can remove this conf and keep things simple.